data/events/scripts/monster.lua

```diff
@@ -0,0 +1,37 @@
+function Monster:onDropLoot(corpse)
+	if (configManager.getNumber(configKeys.RATE_LOOT) == 0) then
+		corpse:decay()
+		return
+	end
+	
+	local player = Player(corpse:getCorpseOwner())
+	local mType = self:getType()
+	if not player or player:getStamina() > 840 then
+		local monsterLoot = mType:getLoot()
+		for i = 1, #monsterLoot do
+			local item = corpse:createLootItem(monsterLoot[i])
+			if not item then
+				print('[Warning] DropLoot:', 'Could not add loot item to corpse.')
+			end
+		end
+
+		if player then 
+			local text = ("Loot of %s: %s"):format(mType:getNameDescription(), corpse:getContentDescription())
+			local party = player:getParty()
+			if party then
+				party:broadcastPartyLoot(text)
+			else
+				player:sendTextMessage(MESSAGE_LOOT, text)
+			end
+		end
+	else
+		local text = ("Loot of %s: nothing (due to low stamina)"):format(mType:getNameDescription())
```

Comment:
Kinda no use for this variable since you only use it once, move them to the function(s) insted.

Reply:
you mean variable text? I was thinking on move it outside.